### DISCUSSION TITLE
Default screen location for annotation options bar

### DISCUSSION THREAD

**Discussion by dhpup** [Softal-io/presentify-community#15] [Ideas]
The bar currently defaults in a place where my video conference tool blocks it. 

Would be awesome to have a few different defaults, or maybe be able to save the location where I move it. 

Right now I have to move the bar to change my annotation method, then when I stop annotating and start it again, the bar resets back to where its hidden for me. 

<img width="610" alt="Screenshot 2024-01-19 at 12 04 42 PM" src="https://github.com/rampatra/presentify-community/assets/21094282/0529cc4b-37ee-4cc2-96df-5873f02ebff3">

**Comment by dhpup** [comment]
I actually just spent some time to learn about shortcuts, and interactive mode today.. I probably don't even need this request anymore.

This still might help some users however lol

**Comment by rampatra** [comment]
That’s good to know but I think it’s still a nice feature to have — a default location for the toolbar. Thanks for requesting.
